json_schemas/report_channelback_error.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "object",
+  "required": ["instance_push_id", "external_id"],
+  "properties": {
+    "instance_push_id": { "type": "string", "minLength": 1, "maxLength": 255 },
```

Comment:
Maybe a dumb question, but why is `"minLength": 1` necessary? Does `required` only checks for the presence of a property, not whether it has a value?

Reply:
Yes, without `"minLength": 1`, the zero length string would be acceptable for `instance_push_id`, but we're saying that the zero length string is not an acceptable ID.  That might be viewed as arbitrary- maybe we SHOULD allow a zero length string?

In practice, the string cannot be zero length, since Zendesk creates the string, and will never make a zero length string.

If the Integration Service passes in a zero length string, that's definitely an error on their part, so this requirement should make that clear, and when we validate their payload with this schema, we'll "fail fast" indicating that the `instance_push_id` is invalid.